example.env

```diff
@@ -0,0 +1,10 @@
+
+# ECSDA Account ID
+OPERATOR_ID_A=
+
+# ECDSA DER Encoded Private Key
+OPERATOR_KEY_A=
+
+# ECDSA HEX Encoded Private Key
+HEX_PRIVATE_KEY_A=
+HEX_PRIVATE_KEY_B=
```

Comment:
Last time I'll mention this - but maybe prettier has a setting?

Reply:
Yes, do we have a description or specification for our code standards?

Reply:
Oh are you guys talking about the `.prettierrc` where we can set the rules? I talked about this [here](https://github.com/hashgraph/hedera-smart-contracts/pull/267#discussion_r1268447447)

Reply:
Maybe you can add an [`.editorconfig`](https://prettier.io/docs/en/configuration.html#editorconfig) - we use that in other repos, and it as an "insert final newline" setting - though I can't find a matching prettier setting in its [schema](http://json.schemastore.org/prettierrc) so who knows if that setting is respected.

Reply:
w.r.t. code style - we have this engineering team standard for Java - [007 - Code Style](https://www.notion.so/swirldslabs/007-Java-Code-Style-a0130a7059b044f68447b9ca3a5242b3?pvs=4) - obviously 95% doesn't apply to this repo but it is _based on_ the Google style guides (with a few changes that might be applicable here, like 120 char line limit).

Google has published _multiple_ style guides [here](https://google.github.io/styleguide/) (and they're even updated from time to time).  There's one for JavaScript that could be adopted, another one for TS.  They're quite nailed down - more than I prefer, but I'm in the minority - and have made some debatable choices (IMO!, again I'm in the minority).  They're a good place to start. 

We might want to look for Solidity coding standards too.

We might want to talk to other people in the company coding JS/TS - the SDK folks, and the DA folks come to mind - see if we want a company-wide policy for uniformity.